mne/viz/epochs.py

```diff
@@ -749,7 +748,19 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
         larger than ``info['lowpass']`` (e.g., a 40 Hz lowpass will result in
         at least a 120 Hz displayed sample rate).
 
-        .. versionadded:: 0.15
+        .. versionadded:: 0.15.0
+    noise_cov : instance of Covariance | str | None
+        Noise covariance used to whiten the data while plotting.
+        Whitened data channels are scaled by ``scalings['whitened']``,
+        and their channel names are shown in italic.
+        Can be a string to load a covariance from disk.
+        See also :meth:`mne.Evoked.plot_white` for additional inspection
+        of noise covariance properties when whitening evoked data.
```

Comment:
, specifically, for inspection of expected values and model violations ... {or so}

Reply:
I'm wary of adding more text to an already long description here. I'll add it to the text for `plot_evoked_white`